src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -627,7 +630,7 @@ uct_ud_mlx5_iface_query(uct_iface_h tl_iface, uct_iface_attr_t *iface_attr)
         return status;
     }
 
-    iface_attr->overhead = 80e-9; /* Software overhead */
+    iface_attr->overhead = UCT_UD_MLX5_IFACE_OVERHEAD; /* Software overhead */
```

Comment:
we need to take  it from config value (in case it is modified for some platform)